aten/src/ATen/core/dispatch/DispatchKeyExtractor.cpp

```diff
@@ -0,0 +1,13 @@
+#include <ATen/core/dispatch/DispatchKeyExtractor.h>
+
+namespace c10 {
+
+void DispatchKeyExtractor::setIsOperatorOverridden(DispatchKey k, bool is_overridden) {
```

Comment:
I find the name a bit confusing since this is not strictly about overriding backend fallbacks, this is also set if a backend doesn't even have a fallback but the operator has a kernel for this key. Maybe a better name is `setOperatorHasKernelForBackend` (and also rename the member variable).